yearn/deploy/01_deploy_convex_strategy.ts

```diff
@@ -0,0 +1,42 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.getOrNull("TBTCCurveVault")
+  if (!TBTCCurveVault || !helpers.address.isValid(TBTCCurveVault.address)) {
+    throw new Error("Address of TBTCCurveVault must be set")
+  }
+
+  const TBTCCurvePoolDepositor = await deployments.getOrNull("TBTCCurvePoolDepositor")
+  if (!TBTCCurvePoolDepositor || !helpers.address.isValid(TBTCCurvePoolDepositor.address)) {
+    throw new Error("Address of TBTCCurvePoolDepositor must be set")
+  }
+
+  const TBTCConvexRewardPool = await deployments.getOrNull("TBTCConvexRewardPool")
+  if (!TBTCConvexRewardPool || !TBTCConvexRewardPool.linkedData.id) {
+    throw new Error("ID of TBTCConvexRewardPool must be set")
```

Comment:
Do we know if `TBTCConvexRewardPool` ID will be actually `0`? If yes, then the error will be thrown, cause `!0` => `1` and we enter inside of the `if` statement.

Reply:
It won't be for sure.